build.gradle

```diff
@@ -29,7 +29,7 @@ buildscript {
                 'mockito2'                : '2.28.2',         // https://github.com/mockito/mockito/releases
                 'mockitoAndroid'          : '3.3.3',          // https://mvnrepository.com/artifact/org.mockito/mockito-android
                 'mockitokotlin'           : '2.1.0',          // https://github.com/nhaarman/mockito-kotlin
-                'testify'                 : '1.0.0-beta4',    // https://github.com/Shopify/android-testify/releases
+                'testify'                 : '1.0.0-beta5',    // https://github.com/Shopify/android-testify/releases
```

Comment:
Is the uptick being included in this PR intentional?

Reply:
Yes